server/protocols/webadmin-cli/README.md

```diff
@@ -0,0 +1,128 @@
+# A WebAdmin based CLI for Apache James
+
+## Development
+
+Webadmin command-line interface is an upcoming replacement for the outdated, security-vulnerable JMX command-line interface. It also aims at providing a more modern and intuitive interface.
+
+## Run the script
+
+## Syntax
+
+General syntax to run the script
```

Comment:
We are doing java, we should remove the word "script" from this document.

Reply:
i replaced "script" by "command line".
By the way, will i need to pull a new request to push new updated README.rm?

Reply:
I pushed new updated README.rm, found out not necessary to pull new request.